dev/TabView/TestUI/TabViewPage.xaml.cs

```diff
@@ -409,6 +419,96 @@ private void ShortLongTextButton_Click(object sender, RoutedEventArgs e)
             FirstTab.Header = "s";
             LongHeaderTab.Header = "long long long long long long long long";
         }
+
+        private void HomeTabOverlapCheck_Click(object sender, RoutedEventArgs e)
+        {
+            var redBrush = new SolidColorBrush();
+            redBrush.Color = Colors.Red;
+            BackgroundGrid.Background = redBrush;
+
+            var tabBrush = new SolidColorBrush();
+            tabBrush.Color = Colors.Blue;
+            SetFirstTabSelectedBackgroundPathFill(tabBrush);
+
+            var contentBrush = new SolidColorBrush();
+            contentBrush.Color = Colors.Green;
```

Comment:
should these colors be translucent?

Reply:
Or maybe have a button to set translucency to 50%

Reply:
I added buttons to set each of these to transparent. Does that cover this use case or was there another use case you are thinking of?